src/braket/circuits/gate.py

```diff
@@ -100,7 +103,7 @@ def to_ir(
             ValueError: If gate modifiers are supplied with `ir_type` Jaqcd.
         """
         if ir_type == IRType.JAQCD:
-            if control:
+            if control or power is not None:
```

Comment:
```suggestion
            if control or power != 1:
```

Reply:
> What about directly implementing `power` in `AngledGate` as `power * angle`?

Open to optimizing this in a follow up, for now I don't mind the consistency of handling all gates the same. Backend could always optimize this as well